parse_test.go

```diff
@@ -860,6 +860,27 @@ func TestEnvironmentVariableInSubcommandIgnored(t *testing.T) {
 	assert.Equal(t, "", args.Sub.Foo)
 }
 
+func TestParserMustParse(t *testing.T) {
+	originalExit := osExit
+	originalStdout := stdout
+	defer func() {
+		osExit = originalExit
+		stdout = originalStdout
+	}()
+
+	var exitCode int
+	osExit = func(code int) { exitCode = code }
+	stdout = &bytes.Buffer{}
+
+	p, err := NewParser(Config{}, &struct{}{})
+	require.NoError(t, err)
+	assert.NotNil(t, p)
+
+	p.MustParse(nil)
+	require.NotNil(t, exitCode)
```

Comment:
`exitCode` is an int - I think according to this line you meant for it to be a `*int`